api/src/main/webapp/WEB-INF/cron_base.yaml

```diff
@@ -16,7 +16,7 @@ cron:
   schedule: every 24 hours
   timezone: UTC
   target: api
-- description: Find and alert users that have exceeded their free tier billing usage using cloud task
+- description: Find and alert users that have exceeded their initial credit usage using cloud task
   url: /v1/cron/checkFreeTierBillingUsage
```

Comment:
minor: do you also want to rename this method?

Reply:
Thanks for brining this up.

This change will result in having to change a few different things, and the cloud task is in need of some larger cleanup, so I am hoping to do that work here:
https://precisionmedicineinitiative.atlassian.net/browse/RW-13494